gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala

```diff
@@ -95,6 +95,10 @@ class RegressPhenotypesArgs extends Args4jBase {
 
   @Args4jOption(required = false, name = "-oneTwo", usage = "If cases are 1 and controls 2 instead of 0 and 1")
   var oneTwo = false
+
+  @Args4jOption(required = false, name = "-writeMissingPheno", usage = "Write missing phenotypes to the location supplied.")
```

Comment:
for better or worse ADAM cli uses `-single_dash_snake_case` for command line arguments